### ISSUE TITLE
Mass refactoring.

### ISSUE THREAD

**Issue by jpittis** [Queens-Hacks/ants#8]
Let's make the code base a little less sketchy?

**Comment by MaxBittker** [comment]
we could do a code review tomorrow :+1:

**Comment by mystor** [comment]
I was looking it over today, and it's not as sketchy as I was expecting it
to be after how little sleep we got. We could certainly improve it in a few
places, though I don't want to go overboard.

**Comment by epilk** [comment]
Sure, lets also make a branch where we can start refactoring and breaking stuff?

**Comment by jpittis** [comment]
Maybe lets talk tomorrow about features so we don't refactor anything we're planning to scrap.

**Comment by epilk** [comment]
yep, come to think about it, merging will be a big pain if we branch and devolpement continues in master. maybe we should just do it iteratively.

**Comment by jpittis** [comment]
Every time you add a feature, make the code base a little nicer than the time before! ;)
